### DISCUSSION TITLE
Create a badge icon in the bottom tab bar

### DISCUSSION THREAD

**Discussion by YoonYoeungJun** [RadekVyM/SimpleToolkit#39] [Q&A]
I wonder how to create a badge icon in the bottom tab bar

**Comment by RadekVyM** [comment]
Hi @YoonYoeungJun,

there are multiple ways of adding custom badges to tabs in the tab bar. One way is to extend `ShellContent` with a bindable property for the badge content and bind that property to the badge. I implemented this approach in the [`Sample.SimpleShellTopBar`](https://github.com/RadekVyM/SimpleToolkit/tree/main/src/Samples/Sample.SimpleShellTopBar) sample app.
